packages/sdk/lib/base.ts

```diff
@@ -488,4 +488,65 @@ export class Itly {
   }
 }
 
+export type ResponseLogger = {
```

Comment:
To keep the core as light as possible can you please move this into a dedicated package and import it into other modules that require it. E.g. `packages/plugin-request-logger`, `import { RequestLoggerPlugin } from `@itly/plugin-request-logger`

Reply:
@falconandy Please ignore the request to move this to a separate package. I think that is overkill. Instead can you make a new folder `sdk/lib/internal` and add RequestLogger.ts there with RequestLoggerPlugin, RequestLogger, etc. You can still export these in `sdk/lib/index` so they can easily be used by Plugins but they won't clutter up base.ts.

Reply:
Done